src/main/jooq/io/github/org/programming/jooq/tables/records/TempbanRecord.java

```diff
@@ -0,0 +1,356 @@
+    }
+
+    @Override
+    public String value3() {
+        return getUserId();
+    }
+
+    @Override
+    public String value4() {
+        return getModeratorId();
+    }
+
+    @Override
+    public DayToSecond value5() {
+        return getDuration();
+    }
+
+    @Override
+    public Instant value6() {
+        return getStart();
+    }
+
+    @Override
+    public String value7() {
+        return getReason();
+    }
+
+    @Override
+    public TempbanRecord value1(Integer value) {
+        setId(value);
+        return this;
+    }
+
+    @Override
+    public TempbanRecord value2(String value) {
+        setGuildId(value);
+        return this;
+    }
+
+    @Override
+    public TempbanRecord value3(String value) {
+        setUserId(value);
+        return this;
+    }
+
+    @Override
+    public TempbanRecord value4(String value) {
+        setModeratorId(value);
+        return this;
+    }
+
+    @Override
+    public TempbanRecord value5(DayToSecond value) {
+        setDuration(value);
+        return this;
+    }
+
+    @Override
+    public TempbanRecord value6(Instant value) {
+        setStart(value);
+        return this;
+    }
+
+    @Override
+    public TempbanRecord value7(String value) {
+        setReason(value);
+        return this;
+    }
+
+    @Override
+    public TempbanRecord values(Integer value1, String value2, String value3, String value4, DayToSecond value5, Instant value6, String value7) {
+        value1(value1);
+        value2(value2);
+        value3(value3);
+        value4(value4);
+        value5(value5);
+        value6(value6);
+        value7(value7);
+        return this;
+    }
+
+    // -------------------------------------------------------------------------
+    // Constructors
+    // -------------------------------------------------------------------------
+
+    /**
+     * Create a detached TempbanRecord
+     */
+    public TempbanRecord() {
+        super(Tempban.TEMPBAN);
+    }
+
+    /**
+     * Create a detached, initialised TempbanRecord
+     */
+    public TempbanRecord(Integer id, String guildId, String userId, String moderatorId, DayToSecond duration, Instant start, String reason) {
+        super(Tempban.TEMPBAN);
+
+        setId(id);
+        setGuildId(guildId);
+        setUserId(userId);
+        setModeratorId(moderatorId);
+        setDuration(duration);
+        setStart(start);
+        setReason(reason);
+    }
+
+    /**
+     * Create a detached, initialised TempbanRecord
+     */
+    public TempbanRecord(io.github.org.programming.jooq.tables.pojos.Tempban value) {
+        super(Tempban.TEMPBAN);
+
+        if (value != null) {
+            setId(value.getId());
+            setGuildId(value.getGuildId());
+            setUserId(value.getUserId());
+            setModeratorId(value.getModeratorId());
+            setDuration(value.getDuration());
+            setStart(value.getStart());
+            setReason(value.getReason());
+        }
+    }
+}
```

Comment:
What is the goal of this file ?

Reply:
This is auto generated by jooq, i.e it basically gets the parameter's in the database file and translates it to code so i can use it in my code.